tests/trace/test_mm_w4a16_fp4_reference_correctness.py

```
@@ -0,0 +1,63 @@
+    }[backend]
+    try:
+        inputs = tpl.init(**shape_kwargs)
+        api = flashinfer.mm_w4a16_fp4(
+            inputs["a"],
+            inputs["b"],
+            inputs["b_descale"],
+            inputs["alpha"],
+            backend=backend,
+            block_size=inputs["block_size"],
+        )
+    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)